docs/FAQ.md

```diff
@@ -190,6 +190,19 @@ In the 0.2 release, AQE is supported but all exchanges will default to the CPU.
 release, running on Spark 3.0.1 and higher any operation that is supported on GPU will now stay on 
 the GPU when AQE is enabled. 
 
+#### Why does my query show as not on the GPU when Adaptive Query Execution is enabled?
+
+When running an `explain()` on a query where AQE is on, it is possible that AQE has not finalized
+the plan.  In this case a message stating `AdaptiveSparkPlan isFinalPlan=false` will be printed at
+the top of the physical plan, and the explain output will show the query plan with CPU operators.
+It is possible to view the plan on the GPU by looking at the details for the job in the Spark UI.
+
+```
+== Physical Plan ==
+AdaptiveSparkPlan isFinalPlan=false
```

Comment:
In your example here isFinalPlan is still false. Do we want to explain that the plan on the UI will be updated as the query runs, and the same issue can happen for any AdaptiveSparkPlan where the isFinalPlan=false

Reply:
Updated the text, please review.